test/functional/p2p-leaktests.py

```diff
@@ -158,6 +160,8 @@ def run_test(self):
         allowed_service_bit5_node.add_connection(connections[5])
         allowed_service_bit7_node.add_connection(connections[6])
 
+        NetworkThread().start()  # Network thread stopped when all previous NodeConnCBs disconnected. Restart it
+
         assert wait_until(lambda: allowed_service_bit5_node.message_count["verack"], timeout=10)
```

Comment:
I've seen this test fail locally with high -j on a system with slow I/O at this assert, I assume this patch will help, though I was unable to reproduce reliably to test :(.